### PR TITLE
Add docker up command for local setup

### PR DESCRIPTION
**Pull request by philipgatzka**

### CODE REVIEW

File: cmd/project/templates/compose.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+
+x-environment-app: &environment-app
+  environment:
+    APP_ENV: prod
+    DATABASE_URL: 'mysql://shopware:shopware@database/shopware'
+    DATABASE_HOST: 'database'
```
Comment (pweyck) [inline cmd/project/templates/compose.yaml]: Not necessary anymore

File: cmd/project/docker_build.go [hunk 2]
```diff
@@ -27,72 +28,110 @@ var dockerBuildCmd = &cobra.Command{
 			return err
 		}
 
-		if shopCfg.Docker.PHP.PhpVersion == "" {
-			projectRoot, err := os.Getwd()
+		err = dumpDockerfile(cmd.Context(), shopCfg)
+		if err != nil {
```
Comment (shyim) [inline cmd/project/docker_build.go]:
```suggestion
		if err := dumpDockerfile(cmd.Context(), shopCfg); err != nil {
```

**Review by pweyck — APPROVED** [review]

File: cmd/project/docker_build.go [hunk 3]
```diff
@@ -27,72 +28,110 @@ var dockerBuildCmd = &cobra.Command{
...
+			return err
+		}
+
+		err = dumpDockerIgnore(shopCfg)
```
Comment (shyim) [inline cmd/project/docker_build.go]: same here

File: cmd/project/templates/compose.yaml [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      replicas: 3
+
+volumes:
+  mysql-data:
+  opensearch-data:
+  files:
+  theme:
+  media:
+  thumbnail:
+  sitemap:
```
Comment (shyim) [inline cmd/project/templates/compose.yaml]: add an empty line

**Review by shyim — CHANGES_REQUESTED** [review]

**Review by shyim — APPROVED** [review]

**Pull request merged**